src/hooks/use-provider-tile-linking.tsx

```diff
@@ -61,18 +64,22 @@ export const useProviderTileLinking = ({
         }
       }
       sharedModelManager.addTileSharedModel(model.content, sharedModel);
+      const sharedTiles = sharedModelManager.getSharedModelProviders(sharedModel);
+      logSharedModelDocEvent(LogEventName.GRAPH_TOOL_LINK, model, sharedTiles);
```

Comment:
This can be used for any tile, so there is no guarantee this is a graph being linked to. So I'd recommend just using a generic event name like TILE_LINK.

Reply:
Right, makes sense.